eloquent/ci-nightly-performance.yaml

```diff
@@ -112,7 +112,7 @@ show_plots:
   Performance Test Results:
   - title: Average Round-Trip Time
     y_axis_label: Milliseconds
-    master_csv_name: plot-6509911700247260954.csv
+    master_csv_name: plot-7413814074193988426.csv
```

Comment:
Since there are already plots for "Average Round-Trip Time" and "Max Resident Set Size" for Eloquent on the buildfarm, lets keep the CSV files the same so we don't lose that data. Only for Eloquent, the dashing ones should remain random.

Reply:
sure